creating-data-assets/modules/telemetry/angle-information.md

```diff
@@ -0,0 +1,199 @@
+
+<!-- @TODO Generate a dark mode version of this image -->
+:::{image} images/circular-vertical.png
+:alt: "Elevation Angle Calculation (Circular) Schematic"
+:width: 100%
+:align: center
+:class: only-light
+:::
+
+To calculate the elevation angle, {math}`\phi`, in the case of the circular angle calculation mode, several steps need to be taken.
+In the first step, the {math}`CameraToObject` vector, which is the vector from the {math}`Camera` to the {math}`Object` (not shown in the figure above), is projected onto the plane {math}`A`, creating the vector {math}`P_{A}(Object)`. The {math}`P_{View}(Object)` vector is the {math}`CameraToObject` vector projected onto the {math}`Camera_{View}` vector (which is the normal of the plane {math}`A`). This gives the formula to calculate the {math}`P_{A}(Object)` vector as follows:
+
+:::{math}
+  P_{A}(Object) = CameraToObject - P_{View}(Object)
+:::
+
+In the next step, the angle {math}`\theta`, between the {math}`Camera_{Up}` vector, and the {math}`P_{A}(Object)` vector, is calculated with the {math}`Camera_{View}` vector (i.e. the normal of the plane {math}`A`) used as the reference axis. The next step is to counter-rotate the {math}`CameraToObject` vector with this angle {math}`\theta` to the {math}`Camera_{Up}` vector. The resulting vector {math}`R_{R}(P_{A}(Object))` is now located within the reference plane {math}`R`.
+
+Lastly, the elevation angle, {math}`\phi`, between the {math}`Camera_{View}` vector, and the {math}`R_{R}(P_{A}(Object))` vector is calculated, with the {math}`Camera_{Right}` vector (i.e. the normal of the reference plane {math}`R`) used as the reference axis. The elevation angle goes from {math}`-\dfrac{\pi}{2}` to {math}`\dfrac{\pi}{2}` in radians, and zero radians would be directly in the {math}`Camera_{View}` direction. When the object is located above the {math}`Camera_{View}` vector, then the elevation angle will be a positive value, as shown in the figure above. If instead, the object is located below, the elevation angle will become negative.
+
+### Moon Angle
+(moon-angle-circular)=
+This angle is only calculated for the specialized [Planets Sonification](./telemetry-types-specialized.md#planets-sonification) telemetry type. It is the angle from the planet, {math}`A`, to one of its moons, {math}`B`, within a circular space around the center of the screen, how this angle is computed is explained below.
+
+The goal is to calculate the angle, {math}`\theta`, from object {math}`A` (the planet) to object {math}`B` (the moon) within a plane of reference, {math}`R`. The reference plane {math}`R` is the plane spanned by the {math}`Camera_{Up}` vector and {math}`Camera_{Right}` vector, with the {math}`Camera_{View}` vector as the normal. The image below shows a figure of these objects and the computations.
```

Comment:
Again, please avoid the phrasing "the goal is". It sounds a bit weird. 

Maybe something like: 

```suggestion
The angle {math}`\theta` is here defined as the angle from object {math}`A` (the planet) to object {math}`B` (the moon) within a plane of reference, {math}`R`, with respect to a direction in relation to the camera. The reference plane {math}`R` is the plane spanned by the {math}`Camera_{Up}` vector and {math}`Camera_{Right}` vector, with the {math}`Camera_{View}` vector as the normal. The image below shows a figure of these objects and the computations.
```